filebeat/harvester/registry.go

```diff
@@ -0,0 +1,69 @@
+package harvester
+
+import (
+	"sync"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+type Registry struct {
+	sync.RWMutex
+	harvesters map[uuid.UUID]Harvester
+	wg         sync.WaitGroup
+}
+
+func NewRegistry() *Registry {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewRegistry should have comment or be unexported